packages/orchestrator-ui-components/src/components/WfoTable/WfoTableWithFilter/updateQueryString.ts

```diff
@@ -0,0 +1,52 @@
+const getValueWithoutBrackets = (value: string) => {
+    if (value.startsWith('(') && value.endsWith(')')) {
+        return value.slice(1, -1);
+    }
+    return value;
+};
+
+const isContainingSpaces = (value: string) => {
+    return value.indexOf(' ') !== -1;
+};
+
+const toQueryValue = (value: string) => {
```

Comment:
IMO quoteSpaces would be more descriptive here because that's all it does.

Reply:
IMO it does not quote te spaces itself. Therefore i went for this more generic name